Neos.Flow/Tests/Functional/Persistence/FixturesPHP8/Post.php

```diff
@@ -0,0 +1,126 @@
+<?php
+namespace Neos\Flow\Tests\Functional\Persistence\FixturesPHP8;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Neos\Flow\Annotations as Flow;
+
+/**
+ * A sample entity for tests
+ */
+#[Flow\Entity]
+#[ORM\HasLifecycleCallbacks]
+#[ORM\InheritanceType('JOINED')]
+class Post
+{
+    protected string $title = '';
+
+    #[ORM\OneToOne]
+    protected Image $image;
+
+    #[ORM\OneToOne]
+    protected Image $thumbnail;
+
+    /**
+     * Yeah, only one comment allowed for a post ;-)
+     * But that's the easiest option for our functional test.
+     */
+    #[ORM\OneToOne]
+    #[ORM\JoinColumn(onDelete: 'SET NULL')]
+    protected Comment $comment;
+
+    /**
+     * @var Collection<Tag>
+     */
+    #[ORM\OneToMany]
+    protected Collection $tags;
+
+    /**
+     * Still annotated because nested Attributes require PHP 8.1
+     * @todo Adjust to attributes for Flow 8.4 when the min PHP version will be 8.1
+     *
+     * @var Collection<Post>
+     * @ORM\ManyToMany
+     * @ORM\JoinTable(inverseJoinColumns={@ORM\JoinColumn(name="related_post_id")})
+     */
+    protected Collection $related;
```

Comment:
shouldn't we now be able to use nested attributes here? From https://github.com/neos/flow-development-collection/pull/3512

Reply:
Yay, tests pass even with nested attributes. 🎉